monocle/db/db.py

```diff
@@ -174,13 +174,38 @@ def __init__(
         create=True,
         previous_schema=False,
         idents_config: Optional[IdentsConfig] = None,
+        user=None,
```

Comment:
So a support for the authentication on the Elastic backend is fine to me. Would you mind to open a single PR with this support in the Monocle API and crawler (only @ python level). No need to provide specific unit test for that.